tests/unit/test-cmd/test.build.js

```diff
@@ -44,6 +45,7 @@ describe('build', () => {
             fileNames.sort();
             assert.deepEqual(fileNames,
                              ['background-script.js', 'manifest.json']);
+            return zipFile.close();
```

Comment:
Ugh, all of these would be way easier with try/catch/finally. We need https://github.com/mozilla/web-ext/issues/755 first though. This was a nasty bug catch, thanks.